### PR TITLE
Add config to allow jsx in classes

### PR DESCRIPTION
**Pull request by noahm**
Our project has some utility classes which are not components but have some methods that return JSX. They don't need to be flagged by this lint rule, and all of our class-based components are already easily identified by the more narrow rules that look for classes extending `React.Component`.

This PR introduces a configuration option to allow disabling that "fuzzier" form of identification for component classes in projects that don't need it.

### CODE REVIEW

**Comment by tatethurston** [comment]
Awesome, thanks @noahm

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-14.16.1
+16.16.0
```
Comment (tatethurston) [inline .nvmrc:1]: ❤️

**Review by tatethurston — APPROVED** [review]

File: .nvmrc (hunk 1 above)
Comment (noahm) [reply .nvmrc:1]: I couldn't actually install deps using 14 due to... reasons

**Comment by noahm** [comment]
I just realized I neglected to update the readme to document the new option. I can take care of that shortly if you'd like

**Comment by tatethurston** [comment]
> I just realized I neglected to update the readme to document the new option. I can take care of that shortly if you'd like

Thanks, I tackled that in #12

**Pull request merged**